python/lsst/pipe/tasks/extended_psf.py

```diff
@@ -566,6 +571,10 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
                         region_name,
                     )
                     continue
+                self.metadata["psfStarCount"][region_name] = 0
+                for brightStarStamps in ref_list:
+                    stamps = brightStarStamps.get()
+                    self.metadata["psfStarCount"][region_name] += len(stamps)
```

Comment:
As you did for `validStarCount` and `allStarCount`, I'd suggest also adding a small in-line comment here to describe what this `psfStarCount` code block is doing. (Both here, and replicated above for the non-regions approach).